docs: docs for distributed cost center (#85)
frappe/erpnext_documentation: erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md

> ---

Reviewer: @KaviyaPeriyasamy add link in contents.md

title: Distributed Cost Center
add_breadcrumbs: 1
show_sidebar: 0

metatags:
 description: In ERPNext, a Distributed cost center enhances the cost center dimension to allow the distribution of the tagged value to a set of cost centers. A distributed cost center can be created out of different ratios of ordinary cost centers and it can be tagged against the GL entries.
 keywords: frappe, erpnext, accounting reports, cost center, GL Entry.
---

<!-- add-breadcrumbs -->
# Distributed Cost Center
 

> The Distributed cost center enhances the cost center dimension to allow the distribution of the tagged value to a set of cost centers. A distributed cost center can be created out of different ratios of ordinary cost centers and it can be tagged against the GL entries. Expense and income will not affect directly to the cost center. It will be distributed with the selected cost centers as 100% equal calculation.

Reviewer: add a brief definition in bold. What this is dist cost center exactly?

> ## 1. How to create the Distributed Cost Center

Reviewer: ```suggestion
## 1. How to create a Distributed Cost Center
```

> 1. Go to the cost center list, click on New.

Reviewer: Cost Center (C caps everywhere in text)

Author: @prasadarr Check my recent commits.

1. Enter the cost center name.
1. Select the parent cost center.
1. Enable the checkbox, **Enable Distributed Cost Center**: On enabling this the distributed cost center table will show, Then select the cost center and allocate the corresponding percentage.
1. Once you are done click on Save.

  <img class="screenshot" alt="Distributed Cost Center" src="{{docs_base_url}}/assets/img/accounts/distributed_cost_center.png">

The following reports will be automatically updated when cost center filter is added:

  * [Accounting Reports](/docs/user/manual/en/accounts/accounting-reports)
    * Financial Statements
    * Budget Variance
    * General Ledger
  * [Profitability Analysis](/docs/user/manual/en/accounts/articles/tracking-project-profitability-using-cost-center)

> ### 2. Related Topic

Reviewer: ```suggestion
### 2. Related Topics
```

---
title: Distributed Cost Center
add_breadcrumbs: 1
show_sidebar: 0

metatags:
 description: A Distributed Cost Center is a Cost Center in which multiple Cost Centers are tagged with an appropriate percentage.
 keywords: frappe, erpnext, accounting reports, Cost Center, GL Entry.
---

<!-- add-breadcrumbs -->
# Distributed Cost Center

**A Distributed Cost Center is a *Cost Center* in which multiple Cost Centers are tagged with an appropriate percentage.**

> 

Reviewer: @KaviyaPeriyasamy can you add an example, new readers will not understand what this, it's purpose

Author: @prasadarr Hope this helps newbie https://github.com/frappe/erpnext_documentation/pull/85/commits/45ef27176197f8d22a69974bdaa8c0bd8e259c78

Reviewer: @KaviyaPeriyasamy it still doesn't tell me what's the benefit of having a distributed cost center. In how are the costs shared? equally? What problem does this feature solve?

Author: @prasadarr Check whether https://github.com/frappe/erpnext_documentation/pull/85/commits/6c0d92b833b448089d96c9fc276a07e1b0f7bb13 it clarifies?

Author: This is the exact thing solved by this feature.

Author: Thanks!

Reviewer: well, just one more thing. What business problem does this solve?
i.e., why would someone want to use this feature?

Author: If a business has a master cost center with dependent cost centers. In every master cost center transactions, it is difficult to update the budget, profit, and loss to each dependent cost center manually with the allocated percentage of the master cost center. This feature helps to automate the process of manual entry.

Reviewer: @KaviyaPeriyasamy 
please add this in the PR, I can't edit your file (tick "Allow edits from maintainers" in future PRs)

Author: Sure. Kindly check this 60d1fcddc082fad73a53db916c1aa09bbe8efd0f

---
title: Distributed Cost Center
add_breadcrumbs: 1
show_sidebar: 0

metatags:
 description: A Distributed Cost Center is a Cost Center in which multiple Cost Centers are tagged with an appropriate percentage.
 keywords: frappe, erpnext, accounting reports, Cost Center, GL Entry.
---

<!-- add-breadcrumbs -->
# Distributed Cost Center

> **A Distributed Cost Center is a *Cost Center* in which multiple Cost Centers are tagged with an appropriate percentage.**

Reviewer: ```suggestion
**A Distributed Cost Center is a Cost Center in which multiple Cost Centers are tagged with an appropriate percentage.**
```

> For example, you can mention the Cost Center 'A' as a Distributed Cost Center. when its income, expense, and budget are shared among various Cost Centers like 'B' and 'C' with some percentage equivalence.

Reviewer: ```suggestion
For example, you can mention the Cost Center 'A' as a Distributed Cost Center when its income, expense, and budget are shared among various Cost Centers like 'B' and 'C' with some percentage equivalence.
```

In ERPNext you can create Distributed Cost Center and use them in transactions and reports.

## 1. How to create a Distributed Cost Center
1. Go to the Cost Center list, click on New.
1. Enter the Cost Center name.
1. Select the parent Cost Center.

> 1. Enable the checkbox, **Enable Distributed Cost Center**: On enabling this the distributed Cost Center table will show, Then select the Cost Center and allocate the corresponding percentage.

Reviewer: ```suggestion
1. Enable the checkbox, **Enable Distributed Cost Center**: On enabling this, the distributed Cost Center table will show. Here, select the Cost Centers and allocate the corresponding percentage.
```

---
title: Distributed Cost Center
add_breadcrumbs: 1
show_sidebar: 0

metatags:
 description: A Distributed Cost Center is a Cost Center in which multiple Cost Centers are tagged with an appropriate percentage.
 keywords: frappe, erpnext, accounting reports, Cost Center, GL Entry.
---

<!-- add-breadcrumbs -->
# Distributed Cost Center

**A Distributed Cost Center is a Cost Center in which multiple Cost Centers are tagged with an appropriate percentage.**

> If a business has a master Cost Center with dependent Cost Centers. In every master Cost Center transactions, it is difficult to update the budget, profit, and loss to each dependent Cost Center manually with the allocated percentage of the master Cost Center. This feature helps to automate the process of manual entry.

Reviewer: ```suggestion
If a business has a master Cost Center with dependent Cost Centers. In every master Cost Center transaction, it is difficult to update the budget, profit, and loss to each dependent Cost Center manually with the allocated percentage of the master Cost Center. This feature helps to automate the process of manual entry.
```

> For example, In your business, If the Cost Center 'B' and 'C' depend on Cost Center 'A' by 20% and 80%. Here, you can mention 'A' as a Distributed Cost Center. It helps to reflect income, expense, and budget of 'A' in 'B' and 'C' with allocated percentages.

Reviewer: ```suggestion
For example, In your business, If the Cost Center 'B' and 'C' depend on Cost Center 'A' by 20% and 80%. Then, you can mention 'A' as a Distributed Cost Center. It helps to reflect income, expense, and budget of 'A' in 'B' and 'C' with allocated percentages.
```
